src/main/kotlin/me/zeroeightsix/kami/feature/module/AutoGG.kt

```diff
@@ -0,0 +1,54 @@
+package me.zeroeightsix.kami.feature.module
+
+import io.github.fablabsmc.fablabs.api.fiber.v1.annotation.Setting
+import me.zero.alpine.listener.EventHandler
+import me.zero.alpine.listener.Listener
+import me.zeroeightsix.kami.event.PacketEvent
+import me.zeroeightsix.kami.gui.text.CompiledText
+import net.minecraft.network.packet.s2c.play.GameMessageS2CPacket
+
+@Module.Info(
+    name = "AutoGG",
+    category = Module.Category.MISC,
+    description = "Says GG or a custom message at the end of a match"
+)
+object AutoGG : Module() {
+    @Setting
+    var ggMessage = CompiledText(mutableListOf())
+
+    private const val updateLimit: Long = 2000L // 2 Seconds
+    private var lastUpdate: Long = 0L
+
+    private val triggers = listOf(
```

Comment:
Add an option to allow the user to customize this?

Reply:
Thats not necessary. You won't get another message that triggers for any more and any less would spam your message.

Reply:
is there really no better way than to do string checks of the chat? this seems inconsistent and unreliable

Reply:
I think it would be better if it was a compiled regex so you can run a single test

Reply:
> I think it would be better if it was a compiled regex so you can run a single test

Regex would probably be slower than checking a few strings.